src/data/set/intervals/basic.lean

```diff
@@ -80,6 +80,19 @@ lemma left_mem_Ici : a ∈ Ici a := by simp
 @[simp] lemma right_mem_Ioc : b ∈ Ioc a b ↔ a < b := by simp [le_refl]
 lemma right_mem_Iic : a ∈ Iic a := by simp
 
+lemma dual_Ici : @Ici (order_dual α) _ a = @Iic α _ a := rfl
```

Comment:
Should these all be simp-lemmas?

Reply:
I don't know. They're useful only if you call something like `convert @dual_lemma (order_dual α) _`, and it fails to unify some intervals because they use different order of arguments to `and_intro`.

Reply:
Oh, ok. I would assume there are all sorts of reason why you could end up with an interval in a dual order.

Reply:
More precisely: I don't care. If I'll be told (without questions) to add `@[simp]`, then I'll do it.

Reply:
I vote for making them `simp`.

Reply:
I've realized that I have to go to bed right now. Could you please push a commit with these `@[simp]` attrs to the `interval-notation` branch?